bedrock/firefox/tests/test_firefox_details.py

```diff
@@ -399,6 +399,25 @@ def test_latest_major_version_no_int(self):
         """latest_major_version should return 0 when no int."""
         eq_(firefox_desktop.latest_major_version('release'), 0)
 
+    def test_funnelcake_direct_links_en_us_win_only(self):
```

Comment:
should now add `@override_settings` for the funnelcake settings. Could also set those to something other than the defaults to prove they work.